VERSION

```diff
@@ -1 +1 @@
-4.0.3
\ No newline at end of file
+4.0.4
```

Comment:
Not certain if this version is updated manually or done automatically during deploy